modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form-report/index.scss

```diff
@@ -53,55 +53,128 @@
 .col-md-6,
 .col-md-8 {
 	.lfr-de__form-report {
-		display: block;
-		.report-cards-area,
+		&--cards-area,
 		.card-item {
+			margin-left: 0;
+			padding-left: 0;
+			padding-right: 0;
 			width: auto;
 		}
 	}
+}
 
-	.report-cards-shortcut {
-		display: none;
+.col-md-4 {
+	.lfr-de__form-report {
+		display: block;
 	}
 }
 
-.col-md-12 {
+.col-md-6 {
 	.lfr-de__form-report {
-		.report-cards-area {
-			width: 100%;
-		}
+		display: block;
+	}
+}
+
+.col-md-8 {
+	.lfr-de__form-report {
+		display: flex;
+	}
+}
+
+//Show collapsed vertical navigation for the smallest widget page sizes
+.col-md-4,
+.col-md-6 {
+	.lfr-de__form-report--cards-shortcut {
+		display: none;
+	}
 
-		.report-cards-shortcut {
-			display: none;
+	.lfr-de__form-report--vertical-nav {
+		display: block;
+
+		.menubar-toggler {
+			display: block;
 		}
+	}
 
-		@media (min-width: 1280px) {
-			.report-cards-area {
-				width: auto;
+	.menubar-vertical-expand-lg.menubar-transparent .menubar-toggler {
+		color: #272833;
```

Comment:
If we have a variable for this color in 'atlas-variables', please change this to keep the pattern we've been doing